src/styles/themes/_default-theme.scss

```diff
@@ -594,8 +609,37 @@
   --button-secondary-disruptive-active-border-color: var(
     --button-secondary-disruptive-active-background-color
   );
+  --button-secondary-disruptive-split-divider-color: var(--disruptive-color-70);
   // ------ Secondary Disruptive Button theme ------
 
+  // ------ System UI Button theme ------
+  --button-system-ui-text-color: var(--grey-color-60);
+  --button-system-ui-background-color: var(--background-color);
+  --button-system-ui-counter-default-text-color: var(--grey-color-80);
+  --button-system-ui-hover-background-color: var(--grey-color-10);
+  --button-system-ui-focus-visible-background-color: var(--background-color);
+  --button-system-ui-active-background-color: var(--grey-color-20);
+  --button-system-ui-split-divider-color: var(--grey-color-70);
+  // ------ System UI Button theme ------
+
+  // ------ Two State Button theme ------
+  --button-two-state-text-color: var(--grey-color-70);
+  --button-two-state-background-color: var(--grey-color-10);
+  --button-two-state-border-color: var(--button-two-state-background-color);
+  --button-two-state-counter-default-background-color: var(--green-color-20);
+  --button-two-state-counter-hover-background-color: var(--green-color-20);
```

Comment:
i find it easier to reason about the groupings if its the button type followed by modified button state so all related styles are together, i.e. `button-two-state-hover-counter-background-color` ... same for active, focus etc. This is approximately the pattern i used for the other buttons so might be cleaner to follow it